README.md

```diff
@@ -265,6 +265,19 @@ Specify the following `oauth.*` properties:
 Some authorization servers don't provide the `iss` claim. In that case you would not set `oauth.valid.issuer.uri`, and you would explicitly turn off issuer checking by setting the following option to `false`:
 - `oauth.check.issuer` (e.g. "false")
 
+You can enforce audience checking, which is an OAuth2 mechanism to prevent successful authentication with tokens that are explicitly issued for use by your resource server.
```

Comment:
```suggestion
You can enforce audience checking, which is an OAuth2 mechanism to prevent successful authentication with tokens that are not explicitly issued for use by your resource server.
```

Reply:
Good catch, I suggest a slightly different fix to keep consistency with the next sentence.
```suggestion
You can enforce audience checking, which is an OAuth2 mechanism to prevent successful authentication unless the tokens are explicitly issued for use by your resource server.
```